server/service/src/sync/translations/master_list.rs

```diff
@@ -9,10 +13,45 @@ use super::{
 #[allow(non_snake_case)]
 #[derive(Deserialize)]
 pub struct LegacyListMasterRow {
-    ID: String,
+    #[serde(rename = "ID")]
+    id: String,
     description: String,
+    #[serde(rename = "isProgram")]
+    is_program: bool,
     code: String,
     note: String,
+    #[serde(rename = "programSettings")]
+    program_settings: Option<LegacyProgramSettings>,
+}
+
+#[derive(Deserialize, Clone)]
+struct LegacyProgramSettings {
+    #[serde(rename = "storeTags")]
+    store_tags: LegacyStoreTagAndProgramName,
+}
+
+#[derive(Deserialize, Clone)]
+struct LegacyStoreTagAndProgramName {
+    #[serde(flatten)]
+    tags: std::collections::HashMap<String, LegacyStoreTag>,
+}
+
+#[derive(Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+struct LegacyStoreTag {
```

Comment:
```suggestion
struct LegacyProgramSettingsStoreTag {
```
I was confused as to why we were using store_tags instead of name tags... Now I see it's in the master_list struct.